Dev/config/FirebaseFunctions.js

```diff
@@ -1,32 +1,38 @@
 //This class will contain all the functions that interact with the react native firebase
 //library
-import firebase from "react-native-firebase";
-import strings from "./strings";
+import firebase from 'react-native-firebase';
+import strings from './strings';
 import _ from 'lodash';
 import moment from 'moment';
+import FeedsScreen from "../src/screens/UniversalClassScreens/FeedsScreen";
+import { TabBarIOS } from "react-native";
```

Comment:
Where are you using these in this function? We should try to decouple the data layer from the UI logic and not include UI logic inside the database functions.

Reply:
Yeah I just realized I wasn't using them but at some point in time i probably imported them to test something.